var num = 5;
var de_num = 0;
try {
    eval("alert('ES6 Exception Handling')");
}
catch (e) {
    console.log("Error : " + e.name);
}
